CHANGELOG.md

```diff
@@ -6,7 +6,9 @@
 
 ### Breaking Changes
 
-_None_
+- Removed support for the deprecated `GHHELPER_ACCESS` in favor of `GITHUB_TOKEN` as the default environment variable to set the GitHub API token. [#420]
+- The `github_client:` parameter (aka `ConfigItem`) is now mandatory for all Fastlane actions that use the GitHub API. [#420]
```

Comment:
- Typo fix
 - Adjusting wording, as in practice, many clients won't provide `github_token: …` as part of the parameters used in their call sites and will instead just rely on `GITHUB_TOKEN` env var being set (e.g. [this call site](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/fastlane/lanes/build.rb#L279-L284) won't require any change for it to continue working)

```suggestion
- The `github_token:` parameter (aka `ConfigItem`)–or using the corresponding `GITHUB_TOKEN` env var to set it–is now mandatory for all Fastlane actions that use the GitHub API. [#420]
```